python/doc/api.md

```diff
@@ -0,0 +1,18 @@
+# Experimental Python Bindings
+
+## callExprString
+
+```python
+nix.callExprString(expression: str, arg)
+```
+Parse a nix expression, then call it as a nix function.
```

Comment:
```suggestion
Parse a Nix expression from the given string, then call it as a Nix function.
```

This needs some more information, such as whether it writes to the store when evaluating derivations and such. Also, how to pass parameters to the evaluator?